build.sbt

```diff
@@ -85,27 +89,28 @@ lazy val commonJvmSettings = Seq(
 
 lazy val commonJsSettings = Seq(
   Global / scalaJSStage := FastOptStage,
-  scalacOptions += {
-    val tagOrHash = {
-      if (isSnapshot.value)
-        sys.process.Process("git rev-parse HEAD").lineStream_!.head
-      else version.value
-    }
-    val a = (LocalRootProject / baseDirectory).value.toURI.toString
-    val g = "https://raw.githubusercontent.com/alonsodomin/cron4s/" + tagOrHash
-    s"-P:scalajs:mapSourceURI:$a->$g/"
-  },
+  scalacOptions ++= (if (scalaVersion.value.startsWith("2.")) Seq {
```

Comment:
this causes bad option error in scala 3 scala.js:  https://github.com/alonsodomin/cron4s/actions/runs/5435520680